Packs/GitLab/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "GitLab",
     "description": "Pack for handling gitlab operations",
     "support": "xsoar",
-    "currentVersion": "2.0.0",
+    "currentVersion": "2.1.0",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
```

Comment:
Can also please fix the marketplaces in the pack metadata? 
should be for XSIAM only (marketplacev2)